modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/StoragePartitionMeta.java

```diff
@@ -72,7 +73,9 @@ public class StoragePartitionMeta extends PartitionMeta {
      * @param versionChainTreeRootPageId Version chain tree root page ID.
      * @param indexTreeMetaPageId Index tree meta page ID.
      * @param gcQueueMetaPageId Garbage collection queue meta page ID.
-     * @param estimatedSize Estimated size of the partition.
+     * @param estimatedSize Estimated number of latest entries in the partition.
```

Comment:
Please improve javadoc in `org.apache.ignite.internal.storage.pagememory.StoragePartitionMetaIo#setEstimatedSize`